bundles/core/org.openhab.core.jsr223/build.properties

```diff
@@ -2,3 +2,4 @@ source.. = src/,
 bin.includes = META-INF/,\
                .,\
                OSGI-INF,\
+               OSGI-INF/jsr223engine.xml,\
```

Comment:
please add '/' to OSGI-INF (line before) rather than referencing each file separately